advisories/2.2.0/BRSA-tjnwgl2a.toml

```
@@ -0,0 +1,16 @@
+[advisory]
+id = "BRSA-tjnwgl2a"
+title = "docker CVE-2024-29018"
+cve = "CVE-2024-29018"
+severity = "high"
+description = "A flaw in the `dockerd` design allowed for a potential data exfiltration from 'internal' networks via authoritative DNS servers. This is because `dockerd` will forward DNS requests to the host loopback device, bypassing the container network namespace's normal routing semantics entirely."
+
+[[advisory.products]]
+package-name = "docker"
```

Need to align these package names with the package name from the spec file:
```suggestion
package-name = "docker-engine"
```